gh-actions/go/code-sanity/action.yaml

```diff
@@ -67,9 +67,13 @@ runs:
           name=$1
           path=$2
           version="$(
-            go mod edit --json | jq -r ".Require[] | select(.Path==\"${path}\") | .Version" || echo 'latest'
+            go mod edit --json | jq -r ".Require[] | select(.Path==\"${path}\") | .Version" || true
```

Comment:
any reason to use || true if it always exit with 0 for a valid input script?

Reply:
That I forgot.